tensorboardX/writer.py

```diff
@@ -18,18 +18,28 @@
 from __future__ import division
 from __future__ import print_function
 
-import time
 import json
 import os
+import six
+import time
+
+try:
+    from caffe2.proto import caffe2_pb2
+    from caffe2.python import cnn, core, workspace
+    from .caffe2_graph import model_to_graph, nets_to_graph, protos_to_graph
+except ImportError:
+    # TODO (ml7): Remove try-except when PyTorch 1.0 merges PyTorch and Caffe2
+    print('Caffe2 is not installed. Disabling Caffe2 functionality')
```

Comment:
We should remove this print. I dont think it's useful for the majority of the PyTorch users